doc/source/tutorials/quickstart/quickstart.rst

```diff
@@ -0,0 +1,59 @@
+===========
+Quick Start
+===========
+This is a short condensed segment aimed at those who want a very quick and dirty rundown on the basics of *igraph*. For a more in depth explanation about what each part of this code is doing, check out `tutorial link.<>`_
+
+.. code-block:: python
+
+    import igraph as ig
+    import matplotlib.pyplot as plt
+
+    # Construct a graph with 3 vertices
+    n_vertices = 3
+    edges = [(0, 1), (1, 2), (2, 0)]
+    g = ig.Graph(n_vertices, edges)
+
+    # Set attributes for the graph, nodes, and edges
+    g["title"] = "Cities of America"
```

Comment:
This will be an absolutely subjective nitpick and I might get shot down, but here it goes ... I thin the English-speaking internet is much too US-centric as it is. People use American examples without thinking all the time even when they are not Americans and have never been there. If you are American and particularly love this example, then please go ahead. Otherwise, can we choose a neutral example please?

Reply:
Your assessment was spot on! I'm not American ( born in Australia :) ) so it also surprises me that I went to a American example without thinking about it. 

I've changed the example to a social network to make it more neutral.